### PR TITLE
dnj challenge - clock

### PR DESCRIPTION
**Pull request by iman-azimi**

### CODE REVIEW

File: challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge/src/assets/main.scss [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    position: absolute;
+    z-index: 2;
+    width: 3px;
+    height: 10px;
+    background: #000;
+    left: 50%;
+    margin-left: -1px;
+    transform-origin: 50% 150px;
+  }
+  .info {
```
Comment (yeganemehr) [inline challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge/src/assets/main.scss]: What are these properties for such an element that does not exist?

File: challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge/src/components/AppClock.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  <section class="app-timer">
+    <div
+      class="app-timer__hours"
+      v-for="hours in activeHours"
+      :key="hours.i"
+      :style="getTimesStyle(hours)"
+    ></div>
+
+    <div class="app-clock">
+      <div class="circle">
```
Comment (yeganemehr) [inline challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge/src/components/AppClock.vue]: this `<div>` and it's children what are doing?

**Review by yeganemehr — CHANGES_REQUESTED** [review]
Hi Iman
Thank you for participating in the challenge!
Your code is so clean and promising.
BTW thanks for netlify deployment, that improve my UX as tester.
Can you move your code from `challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge` to `challenges/2/YOUR-CODE-GOES-HERE/iman-azimi` so I can merge it?

File: challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge/src/assets/main.scss (hunk 1 above)
Comment (iman-azimi) [reply challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge/src/assets/main.scss]: Unfortunately, I forgot to delete some items that were created for the test..I will do these items as soon as possible

**Comment by iman-azimi** [comment]
> Hi Iman Thank you for participating in the challenge! Your code is so clean and promising. BTW thanks for netlify deployment, that improve my UX as tester. Can you move your code from `challenges/2/YOUR-CODE-GOES-HERE/dnj-challenge` to `challenges/2/YOUR-CODE-GOES-HERE/iman-azimi` so I can merge it?


Thank you for your positive opinion..I won't have access to do this tomorrow due to some problems

**Comment by iman-azimi** [comment]
> _ No description provided. _

Hello, Mr.YaganehMehr.
The things you said have been fixed!

Sorry for being late.

**Review by yeganemehr — CHANGES_REQUESTED** [review]
Hi Iman
Thank you for apply new changes.
But the code still is in wrong directory.
please place your work in `challenges/2/YOUR-CODE-GOES-HERE/iman-azimi` directory.

**Comment by iman-azimi** [comment]
> Hi Iman Thank you for apply new changes. But the code still is in wrong directory. please place your work in `challenges/2/YOUR-CODE-GOES-HERE/iman-azimi` directory.

Sorry
I got it wrong
This is resolved

**Review by yeganemehr — APPROVED** [review]

**Comment by yeganemehr** [comment]
Thanks for your cooperation.
HR will call soon.

**Comment by iman-azimi** [comment]
> Thanks for your cooperation. HR will call soon.

Thanks for your cooperation

**Pull request merged**